inspirehep/modules/disambiguation/config.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Inspire.
+# Copyright (C) 2016 CERN.
+#
+# Inspire is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Inspire is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Inspire; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Configuration of Disambiguation module."""
+
+from __future__ import absolute_import, division, print_function, \
+    unicode_literals
+
+from datetime import timedelta
+
+# Celery.
+CELERYBEAT_SCHEDULE = {
+    'run-beard-every-24h': {
+        'task': 'inspirehep.modules.disambiguation.tasks.disambiguation_demon',
+        'schedule': timedelta(hours=24)
+    },
+}
+DISAMBIGUATION_QUEUE = "beard"
+
+# Elasticsearch.
+HEP_ELASTICSEARCH_INDEX = "records-hep"
+HEP_ELASTICSEARCH_TYPE = "hep"
```

Comment:
I would rather have a variable:

DISAMBIGUATION_RECORD_TYPES = ["hep"]
(and derive the index from that), so that we can plug more than one type of record (e.g. soon "Data").